src/compiler/checker.ts

```diff
@@ -2121,12 +2121,16 @@ namespace ts {
             }
 
             function buildTypeDisplay(type: Type, writer: SymbolWriter, enclosingDeclaration?: Node, globalFlags?: TypeFormatFlags, symbolStack?: Symbol[]) {
-                const globalFlagsToPass = globalFlags & TypeFormatFlags.WriteOwnNameForAnyLike;
+                let globalFlagsToPass = globalFlags & TypeFormatFlags.WriteOwnNameForAnyLike;

```

Comment:
then that will make we serialize type-alias all the time. THough we won't want to do that for quickinfo I think. because in if we want to serialize type-alias all the time, we can just remove the check completely